src/index.js

```diff
@@ -0,0 +1,178 @@
+let temperatureLabel = null;
+let temperatureIncreaseButton = null;
+let temperatureDecreaseButton = null;
+let getRealtimeTemperatureButton = null;
+let skyImage = null;
+let landscapeImage = null;
+let cityNameInput = null;
+let cityNameResetButton = null;
+let skySelectValue = null;
+let gardenBackground = null;
+let temperatureValue = null;
```

Comment:
As with the `temperatureValue` being used to track the temperature without being tied directly to a part of the UI, consider including variables for the selected sky and city name rather than using the `value` properties on their input controls directly. It's often useful to separate the internal value for something we're tracking from a particular input control, making it more flexible in how we set/calculate the values.